django/db/models/sql/query.py

```diff
@@ -1643,6 +1643,8 @@ def add_ordering(self, *ordering):
         for item in ordering:
             if not hasattr(item, 'resolve_expression') and not ORDER_PATTERN.match(item):
                 errors.append(item)
+            if getattr(item, 'contains_aggregate', False):
+                raise FieldError('Direct usage of aggregates in order_by is not allowed: %s' % item)
```

Comment:
is this message an accurate improvement?
`Using an aggregate in order_by() without also including it in annotate() is not allowed:`

I'll also update the test to use `assertRaisesMessage` as suggested by Simon when merging.